bin/run

```diff
@@ -38,6 +62,10 @@ case $1 in
   web)
     airflow initdb
     airflow upgradedb
+
+    # Only init connections in dev
+    [ ! -z ${DEVELOPMENT+check} ] && init_connections
```

Comment:
Could we split this out into another file and source it instead? It'll be easier to point someone to `bin/connections.sh` since it'll be single purposed.

Reply:
I attempted that, but it never worked. I think we can follow-up with that change as it will take some time to figure out why it wasn't running.

Reply:
Created in issue: https://github.com/mozilla/telemetry-airflow/issues/548